### PR TITLE
COM-5628- css changes for item list display

### PR DESCRIPTION
**Pull request by kibo-himanshivarshney**
COM-5628- changes for item list display after downloading the mail.

### CODE REVIEW

File: templates/modules/common/subscription-item-list.hypr.live [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
                     <span>
                         {{item.quantity}}
                     </span>
                 </td>
-                <td>
+                <td style="text-align: right;padding: 15px;">
                     {% include "modules/common/item-price" %}
                 </td>
-                <td>
+                <td style="text-align: right;padding: 15px;">
```
Comment (kibo-prakharkhandelwal) [inline templates/modules/common/subscription-item-list.hypr.live:30]: Don't add inline styling. Create a class. There are `.less` files under spreadsheets. Create one for subscription if needed. Refer how we are doing for orders. `order-listing.less` or `order-summary.less`

**Review by kibo-prakharkhandelwal — CHANGES_REQUESTED** [review]
Don't add inline styling. Create a class. There are `.less` files under spreadsheets. Create one for subscription if needed. Refer how we are doing for orders. `order-listing.less` or `order-summary.less`

File: templates/modules/common/subscription-item-list.hypr.live (hunk 1 above)
Comment (kibo-himanshivarshney) [reply templates/modules/common/subscription-item-list.hypr.live:30]: used inline styling for the outlook view of the mails as css classes css won't apply in that view.

**Review by kibo-prakharkhandelwal — APPROVED** [review]

**Comment by kibo-himanshivarshney** [comment]
> Don't add inline styling. Create a class. There are `.less` files under spreadsheets. Create one for subscription if needed. Refer how we are doing for orders. `order-listing.less` or `order-summary.less`

added these inline style css for outlook view.

**Pull request merged**